packages/fx-core/tests/plugins/solution/solution.provision.test.ts

```diff
@@ -500,10 +500,10 @@ describe("provision() happy path for SPFx projects", () => {
   it("should succeed if app studio returns successfully", () =>
     provisionSpfxProjectShouldSucceed(false));
 
-  it("should succeed if arm support feature flag enabled", () =>
+  it.skip("should succeed if insider feature flag enabled", () =>
```

Comment:
@nliu-ms  As discussed offline, this test case is skipped temporary. Please help enable this test case again after you helps fix this.